test/shopify-cli/project_test.rb

```diff
@@ -29,11 +29,13 @@ def test_current_fails_if_no_config
     end
 
     def test_write_writes_yaml
-      create_empty_config
       Shopifolk.stubs(:acting_as_shopify_organization?).returns(false)
+      Dir.stubs(:pwd).returns(@context.root)
+      FileUtils.touch(".shopify-cli.yml")
```

Comment:
This is slightly curious. Is the fact that the test is touching the YAML file a problem, since the test is expected to write that file?

My concern is whether this would potentially mask an issue where we don't write the file if it doesn't exist.

Reply:
This wasn't needed, removed in 73ec07d4.